update_metadata.py

```diff
@@ -0,0 +1,82 @@
+import click
+import os
+import subprocess
+import tempfile
+from bris_fiab.checkpoint.metadata import update_metadata
+
+description = '''
+A command-line tool to update a YAML metadata file with values from another YAML file.
+
+The script loads both YAML files, traverses the specified path, replaces the value in the original
+metadata with the value from the update file, and writes the result to the output file.
+'''
+default_update_metadata_file = 'etc/checkpoint_metadata_part.yaml'
+default_replace_path = 'dataset.variables_metadata'
+
+
+@click.command(help=description)
+@click.option('--checkpoint', type=click.Path(exists=True),  required=True,
+              help='Path to checkpoint to update metadata.')
+@click.option('--update-with-metadata', type=click.Path(exists=True), required=True, default=f"{default_update_metadata_file}",
+              help=f'Path to the YAML file containing updated metadata values. Default: {default_update_metadata_file}')
+@click.option('--replace-path',  type=str, default=f'{default_replace_path}',
```

Comment:
What is the purpose of this? Will it ever be anything else that the default?

Reply:
If we later want to update other metadata.